internal/kafka/internal/handlers/admin_kafka.go

```diff
@@ -202,7 +203,7 @@ func (h *adminKafkaHandler) Update(w http.ResponseWriter, r *http.Request) {
 				return kafka.Status
 			}
 
-			requestedStorageSize, _ := arrays.FirstNonEmpty(kafkaUpdateReq.MaxDataRetentionSize, kafkaUpdateReq.DeprecatedKafkaStorageSize)
+			requestedStorageSize, _ := arrays.FirstNonEmpty(kafkaUpdateReq.MaxDataRetentionSize)
```

Comment:
I think we don't need this anymore. instead we can use `kafkaUpdateReq.MaxDataRetentionSize` directly below, e.g. instead of updateRequired = update(&kafkaRequest.KafkaStorageSize, requestedStorageSize) || updateRequired` to use `updateRequired = update(&kafkaRequest.KafkaStorageSize, kafkaUpdateReq.MaxDataRetentionSize) || updateRequired`

Reply:
looking more into this. @machi1990 - should the `KafkaStorageSize` in the `internal/kafka/internal/api/dbapi/kafka_request_types.go` be changed as well to `MaxDataRetentionSize`?

Reply:
> looking more into this. @machi1990 - should the `KafkaStorageSize` in the `internal/kafka/internal/api/dbapi/kafka_request_types.go` be changed as well to `MaxDataRetentionSize`?

+1. We'll also need a db migration to rename the database column from `kafka_storage_size` to `max_data_retention_size`.